src/Components/Textarea/Textarea.js

```diff
@@ -0,0 +1,30 @@
+import React, { Component } from "react";
+import styles from "./Textarea.module.css";
+import ErrorMessage from "../ErrorMessage/ErrorMessage";
+
+class Textarea extends Component {
+  render() {
+    return (
+      <label className={styles.label}>
+        <div className={styles.title}>
+          <div>{this.props.label}</div>
+          <div className={styles.counter}>
+            {this.props.value?.length > 600 ? 0 : 600 - this.props.value.length}
```

Comment:
You can extract 'magic number' 600 to constant in this file, too.

Reply:
done